python/cuml/cuml/manifold/umap.pyx

```diff
@@ -434,7 +435,18 @@ class UMAP(UniversalBase,
         self.precomputed_knn = extract_knn_infos(precomputed_knn,
                                                  n_neighbors)
 
-        logger.set_level(verbose)
```

Comment:
This logic was previously incorrect because it was propagating boolean verbosity values directly without accounting for the boolean handling done in Base's constructor.